Jenkinsfile

```diff
@@ -1,6 +1,10 @@
 #!/usr/bin/env groovy
 
+DEFAULT_PUBLISHING_E2E_TESTS_BRANCH = "master"
+
 node {
   def govuk = load '/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy'
-  govuk.buildProject()
+  govuk.setEnvar ("PUBLISHING_E2E_TESTS_APP_PARAM", "CONTENT_STORE_COMMITISH")
+  govuk.setEnvar ("PUBLISHING_E2E_TESTS_BRANCH", DEFAULT_PUBLISHING_E2E_TESTS_BRANCH)
+  govuk.buildProject(publishingE2eTests: true)
```

Comment:
I think we only need this line here as it should work out the branch and app param automatically?